pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/TNCudaBase.hpp

```diff
@@ -244,6 +246,87 @@ class TNCudaBase : public TensornetBase<PrecisionT, Derived> {
         }
     }
 
+    /**
+     * @brief Append a single controlled gate tensor to the compute graph.
+     *
+     * NOTE: This function does not update the quantum state but only appends
+     * gate tensor operator to the graph. The controlled gate should be
+     * immutable as v24.08.
+     *
+     * @param baseOpName Base gate's name.
+     * @param controlledWires Controlled wires for the gate.
+     * @param controlled_values Controlled values for the gate.
+     * @param targetWires Target wires for the gate.
+     * @param adjoint Indicates whether to use adjoint of gate.
+     * @param params Optional parameter list for parametric gates.
+     * @param gate_matrix Optional gate matrix for custom gates.
+     */
+    void
+    applyControlledOperation(const std::string &baseOpName,
+                             const std::vector<std::size_t> &controlledWires,
```

Comment:
Are we using lower-camel or snake case for function arguments?
```suggestion
                             const std::vector<std::size_t> &controlled_wires,
```